tests/cmake/CMakeLintBearTest.py

```diff
@@ -6,6 +6,9 @@
 
 good_file = 'project(FooBar C)\nset(VERSION 0)\n'
 
+test_file = ('target_compile_options(example ' +
+             'PUBLIC -std=c++1y -Wall -Wfloat-conversion)')
```

Comment:
this is only 75 chars, which means it is not testing the line length check at all